docker/armorblox/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+armorblox-sdk = "*"
```

Comment:
I think we should fix this to a specific version of the package - we are at 0.1.0 I think.

Reply:
Although I see that the Pipfile.lock has it fixed at 0.1.0, in which case we might not need to do anything.

Reply:
Will let it remain as is